src/mocsolver.jl

```diff
@@ -336,18 +421,23 @@ function add_q_to_φ!(sol::MoCSolution, prob::MoCProblem)
     @inbounds for i in 1:NRegions
         xs = getxs(prob, i)
         @unpack Σt = xs
-
+        if volumes[i] < 1e-12 # deal with zero volume FSRs
+            volumes[i] = 1e30
+        end
         for g in 1:NGroups
             ig = @region_index(i, g)
             φ[ig] /= (Σt[g] * volumes[i])
-            φ[ig] += (4π * q[ig])
+            φ[ig] += (4π * q[ig]) / Σt[g]
```

Comment:
did you changed this line to take into consideration what you remove above? I mean, to take into consideration that you deleted `qig /= (4π * Σt[g])`?

If this is the case, it seems that a `4π` factor is missing somewhere

Reply:
Yes that's correct. I believe you're right that I introduced a mistake. I changed this back to how you had it originally.